server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileRepository.java

```diff
@@ -59,6 +60,14 @@ List<MemberProfile> search(@Nullable String firstName, @Nullable String middleNa
                                @Nullable String workEmail, @Nullable String supervisorId, @Nullable Boolean terminated);
 
     List<MemberProfile> findAll();
+    List<UUID> findSupervisoridByIdIn(Set<UUID> Ids);
+
+    @Query(value = "SELECT " +
+            "PGP_SYM_DECRYPT(cast(workEmail as bytea), :aesKey) as workEmail " +
+            "FROM \"member_profile\" mp " +
+            "WHERE  mp.id IN (:ids) ",
+            nativeQuery = true)
```

Comment:
We can use Java 15 code blocks which I think are easier to read here? WDYT?

```suggestion
    @Query(value = """
            SELECT PGP_SYM_DECRYPT(cast(workEmail as bytea), :aesKey) as workEmail
                    FROM member_profile mp
                    WHERE  mp.id IN (:ids)""",
            nativeQuery = true)
```